src/main/java/com/epam/flightscontroltool/controller/dto/flight/FlightDto.java

```diff
@@ -0,0 +1,65 @@
+package com.epam.flightscontroltool.controller.dto.flight;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.validation.constraints.NotNull;
+import java.math.BigDecimal;
+
+@Getter
+@Setter
+@NoArgsConstructor
+public class FlightDto {
+
+    @ApiModelProperty(
+            value = "flight id",
+            dataType = "long")
+    @JsonProperty("id")
+    private long id;
```

Comment:
Will this value always be filled in? Or what will be the meaning for 0 value?

Reply:
changed to Long